src/mscorlib/src/System/Lazy.cs

```diff
@@ -322,147 +528,16 @@ public T Value
         {
             get
             {
-                Boxed boxed = null;
-                if (m_boxed != null )
+                while (true)
```

Comment:
Can't just return _value without check if _state (volatile semantics). I'm not sure I can guarantee that it would always be valid (although I'm sure it would be in tests, but that's not the same thing is it 😀) I'll return it to how it was.